09-redux/classworks/Shevtsov_Vladimir/src/hooks/useShowSubreddit.js

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+
+const BASE_URL = 'https://oauth.reddit.com';
+
+export const useShowSubreddit = ({ url, apiToken }) => {
```

Comment:
DRY, you have 3 almost same files (hooks). can we reuse the code?

Reply:
fixed